tests/test_polaraveraging.py

```diff
@@ -90,21 +85,15 @@ def test_compute_lensing_angles_flatsky():
     ra_l, dec_l = 161.32, 51.49
     ra_s, dec_s = np.array([161.29, 161.34]), np.array([51.45, 51.55])
     thetas, phis = pa._compute_lensing_angles_flatsky(ra_l, dec_l, ra_s, dec_s)
+
     testing.assert_allclose(thetas, np.array([0.00077050407583119666, 0.00106951489719733675]),
```

Comment:
I'm a bit confused here. Can I ask where these numbers came from? Are they from some special cases? Thanks.

Reply:
I didn't write those tests, but I imagine that they are just sanity tests.